Example/Example.iOS/AppDelegate.cs

```diff
@@ -23,7 +25,7 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
             global::Xamarin.Forms.Forms.Init();
             var xamApp = new App();
-            MobileCoreIOS.Init(xamApp.GetType().Assembly);
+            MobileCoreIOS.Init(xamApp.GetType().Assembly, new Type[] { typeof(AuthService)});
```

Comment:
Does it makes sense to pass the type here? What if we would pass an instance? WDYT?

Reply:
It that case we can just call `new AuthIOS(core)` and omit `GetInstance<>`. So @wtrocki will be happy immensely. :)

Reply:
@wojta : I think the idea behind the `MobileCore.GetInstance` methods was to avoid having multiple instances of services, that is the reason why I was just passing the type here.

Reply:
@ziccardi This is actual feature. I think that we could have services that are defined by user and they needs to have more than one instance.